assets/timber.scss.liquid

```diff
@@ -1190,6 +1190,79 @@ th, td {
   border: 1px solid $colorBorder;
 }
 
+/*============================================================================
+  Responsive tables, defined with .table--responsive on table element.
+  Only defined for IE9+
+==============================================================================*/
+html:not(.lt-ie9) {
+  .table--responsive {
```

Comment:
With respond.js all media queries are processed too. First time it wasn't really wanted.